src/clarity-angular/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -171,54 +121,95 @@ $vertical-nav-divider-margin: $clr_baselineRem_0_5;
-        .nav-link {
+        .nav-trigger,
+        .nav-group-trigger {
+            //Display
+            flex: 0 0 $vertical-nav-item-height;
+
             //Dims
+            border: none;
             height: $vertical-nav-item-height;
-            padding: 0 $vertical-nav-link-padding-right 0 $vertical-nav-link-padding-left;
+            padding: 0;
 
             //Others
-            line-height: $vertical-nav-item-height;
+            background-color: transparent;
+            cursor: pointer;
             outline-offset: -5px;
         }
 
-        .nav-link {
+        .nav-trigger {
             //Display
             display: flex;
+            align-items: center;
+            //Flex-start because of IE11 + margin-left: auto
+            justify-content: flex-start;
+
+            //Dims
+            height: $vertical-nav-item-height;
+            margin-top: -1 * $vertical-nav-top-padding;
+        }
+
+        //Parent .nav-group-content
+        .nav-group-trigger {
+            //Display
+            //flex: 0 0 $vertical-nav-item-height;
+            display: flex;
+            flex: 1 1 auto;
 
             //Others
-            @include vertical-nav-link-colors();
+            color: inherit;
+            overflow: hidden;
+            text-align: left;
+
+            .nav-group-trigger-icon {
+                //Display
+                flex-shrink: 0;
+
+                //Dims
+                height: $vertical-nav-item-height;
+                width: $vertical-nav-icon-size;
+                margin-left: $vertical-nav-trigger-icon-align-margin;
+                margin-right: $vertical-nav-trigger-icon-align-margin;
 
-            &:hover .nav-icon {
-                fill: $clr-action-blue;
+                //Others
+                transition: all 0.2s ease-in-out;
             }
         }
 
-        & > .nav-link,
-        .nav-content > .nav-link {
-            //Display
-            flex: 0 0 $vertical-nav-item-height;
+        //Parent .nav-trigger
+        .nav-trigger-icon {
+            //Dims
+            margin-left: auto;
+            margin-right: ($vertical-nav-item-height - $vertical-nav-icon-size) / 2;
+
+            //Others
+            transition: all 0.2s ease-in-out;
         }
 
-        .nav-icon + .nav-group-text {
-            padding-left: 0;
+        .nav-trigger + .nav-content {
+            //Dims
+            border-top: 1px solid rgba($gray-darker, 0.2);
+            padding-top: $clr_baselineRem_0_5;
         }
 
-        .nav-header {
+        .nav-group-text,
+        .nav-link {
             //Dims
+            height: $vertical-nav-item-height;
             padding: 0 $vertical-nav-link-padding-right 0 $vertical-nav-link-padding-left;
 
             //Others
-            @include clr-getTypePropertiesForDomElement(vertical_nav_header, (font-size, font-weight, letter-spacing));
             line-height: $vertical-nav-item-height;
+            outline-offset: -5px;
```

Comment:
I've noticed this `-5px` for focus in a lot of places in our SCSS. We may want to consider making it a SCSS var at some point. Nothing for you to do here. Just want to know if you think it is a good idea.

If so, I will take care of it with the rem units refactor.

Reply:
Yes this definitely is a good idea. We have it defined in all places which use a link in a component which has `overflow: hidden` & `overflow: auto` because in this case the component clips the outline styles. We should definitely do this later on.